app.py

```diff
@@ -174,6 +187,29 @@ def main():
             capture_exception(e_exception)
             return jsonify({"status": False, "debugMessage": "Request Failed", "error": str(e_exception), "start_time": start_time, "total_time_taken": total_time.seconds, 'version': version, 'code': '2'})
 
+    if request.args.get('method') == "advocatecasesbynumber":
+        try:
+            start_time = datetime.datetime.now()
+            body = request.json
+            params = request.args
+            chrome_driver = create_driver(__location__=None)  # open browser
+            nclt_props = {
+                "driver": chrome_driver,
+                "bench_id": body["benchId"],
+                "case_type_id": body['caseTypeId'],
+                "case_num": body["caseNum"],
+                "case_year": body["caseYear"]
+            }
+            data = get_nclt_data(nclt_props)
+            end_time = datetime.datetime.now()
+            total_time = end_time - start_time
+            return jsonify({"status": True, "data": data, "request": {"body": body, "params": params, "start_time": start_time.isoformat(), "time": total_time.seconds, 'version': version}})
+        except Exception as e_exception:
+            end_time = datetime.datetime.now()
+            total_time = end_time - start_time
+            capture_exception(e_exception)
+            return jsonify({"status": False, "debugMessage": "Request Failed", "error": str(e_exception), "start_time": start_time, "total_time_taken": total_time.seconds, 'version': version, 'code': '3'})
```

Comment:
errors code should be unique across the app, not for just 1 method. If there is 3 somewhere else then it shouldnt be here